tests/test_flow/test_persistence_compatability.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from .generate_test_compatability_cache import Harness
+
+
+@pytest.fixture
+def older_serialized_cache_harness(builder, datadir):
+    harness = Harness()
+    harness.builder.set('core__persistent_cache__flow_dir', datadir)
```

Comment:
Very minor thing, but WDYT about passing the directory as an argument to `Harness` rather than updating the builder object?

Reply:
This makes sense to me. The test evolved a lot and some of the things are not really needed. I'm also going to get rid of pytest-datadir as we discussed in person last week since that also does not seem to be needed anymore.